packages/validate/test/ValidateMixin.test.js

```diff
@@ -1030,19 +1055,50 @@ describe.only('ValidateMixin', () => {
     ContainsLowercaseA.getMessage = () => 'Message for containsLowercaseA';
     ContainsCat.getMessage = () => 'Message for containsCat';
 
+    it('sets ".hasErrorVisible"/[has-error-visible] when visibility condition is met', async () => {
+      const el = await fixture(html`
+        <${tag} .validators=${[new MinLength(3)]}>${lightDom}</${tag}>`);
+
+      if (cfg.enableFeedbackVisible) {
+        cfg.enableFeedbackVisible(el);
+      }
+
+      el.modelValue = 'a';
+      await el.feedbackComplete;
+      expect(el.hasErrorVisible).to.be.true;
+      expect(el.hasAttribute('has-error-visible')).to.be.true;
+
+      el.modelValue = 'abc';
+      await el.feedbackComplete;
+      expect(el.hasErrorVisible).to.be.false;
+      expect(el.hasAttribute('has-error-visible')).to.be.false;
+    });
+
+    it('passes a message to the "._feedbackNode"', async () => {
+      const el = await fixture(html`
+        <${tag}
+          .modelValue=${'cat'}
+        >${lightDom}</${tag}>
+      `);
+      expect(el._feedbackNode.message).to.equal('');
+      el.validators = [new AlwaysInvalid()];
+      await el.feedbackComplete;
+      expect(el._feedbackNode.message).to.equal('Message for alwaysInvalid');
+    });
+
     it('has configurable feedback visibility hook', async () => {
       const el = await fixture(html`
         <${tag}
-          ._prioritizeAndFilterFeedback=${() => []}
```

Comment:
so we do not want to allow overriding it within the template? e.g. only when extending? then we should also not test it via
```
el._prioritizeAndFilterFeedback = () => [];
```
?